aviary/docs/user_guide/aviary_commands.ipynb

```diff
@@ -447,6 +488,72 @@
    "source": [
     "!aviary convert_prop_table -h"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-draw-command)=\n",
+    "### aviary draw_mission\n",
+    "\n",
+    "The `aviary draw_mission` command will bring up a new graphical interface for users to create a  mission. This command does not have an input. More details can be found at [Drawing and running simple missions](drawing_and_running_simple_missions.ipynb)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-plot-command)=\n",
+    "### aviary plot_drag_polar\n",
+    "\n",
+    "The `aviary draw_mission` command will bring up a new graphical interface for users to draw drag polar. No option is needed on the command line but a windows explorer will pop-up to select a drag polar file (a `.csv` file). It is not working at this time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!aviary plot_drag_polar -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "(aviary-dashboard-command)=\n",
+    "### aviary dashboard\n",
+    "\n",
+    "The `aviary dashboard` command will bring up a dashboard that lets the user easily browse between the reports and files that are generated during an Aviary run."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!aviary dashboard -h"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To use this utility, either a problem has been run or a run script is provided.\n",
+    "\n",
+    "`--problem_recorder` is an input. Default is `problem_history.db`.\n",
+    "`--driver_recorder` is an optional input.\n",
+    "`--port` is the dashboard server port ID. The default is `0` meaning any free port.\n",
+    "`-b` or `--background` indicates to run in background. Default is `False`.\n",
+    "`-d` or `--debug` indicates to show debugging output. Default is `False`.\n",
+    "`--save` is the name of zip file in which dashboard files are saved. If no argument given, use the script name to name the zip file.\n",
+    "`--force` indicates to overwrite the saved zip file. The default is `False` (TODO: check with Herb. It does not have impact).\n",
```

Comment:
```suggestion
    "`--force` indicates to overwrite the saved zip file. The default is `False` (TODO: Confirm that it does not have an impact).\n",
```

Reply:
I checked with Herb. He confirmed that it is correct. So, I removed `TODO` part.